### ISSUE TITLE
Regarding training of HRSCD3 and HRSCD4

### ISSUE THREAD

**Issue by AishaJaved123** [long123524/HGINet-torch#6]
Hello, 

Thank you for your code as well as the codes for the networks used during comparative analysis in your paper.
I have a question regarding the learning rate and optimizer used for training HRSCD techniques. Have you set them similarly to the training strategy of your network or have you used other hyperparameters? If you used another then can you please share it here for each dataset while training HRSCD3 and 4? 

Thank you
Looking forward to hearing from you soon

**Comment by long123524** [comment]
> Hello,
> 
> Thank you for your code as well as the codes for the networks used during comparative analysis in your paper. I have a question regarding the learning rate and optimizer used for training HRSCD techniques. Have you set them similarly to the training strategy of your network or have you used other hyperparameters? If you used another then can you please share it here for each dataset while training HRSCD3 and 4?
> 
> Thank you Looking forward to hearing from you soon

Thanks. The HRSCD series referring to the existing setting, i.e., using the SGD opimizer with initial learning of 0.1. In recent studies, we also find that using the Adam optimizer with initial learning of 0.0001 is good. You can also try it.
